imap/lmtp_sieve.c

```diff
@@ -2120,6 +2120,14 @@ static int send_response(void *ac, void *ic,
 
         prometheus_increment(CYRUS_LMTP_SIEVE_AUTORESPOND_SENT_TOTAL);
 
+        syslog(LOG_INFO, "sieve autoresponded: %s to: %s",
+               md->id ? md->id : "<nomsgid>", src->addr);
+        if (config_auditlog) {
+            syslog(LOG_NOTICE,
```

Comment:
might want to use xsyslog here, didn't we deprecate syslog in favor of it?

Reply:
Done.